app/views/users/show.html.slim

```diff
@@ -33,7 +33,8 @@
               = t(@user.subscription.state)
             li
               ' Última renovação:
-              = @user.subscription.activated_at
+              - @user.subscription.activated_at.present?
```

Comment:
Two things:

- Aren't you missing an `if` here?
- I think the check should come before the `li` tag. That way we only show the item when there is something to show ;)